app/serializers/resource_serializer.rb

```diff
@@ -1,10 +1,20 @@
 # frozen_string_literal: true
 
 class ResourceSerializer < ActiveModel::Serializer
+  type 'resource'
   attributes :id, :name, :abbreviation, :onesky_project_id
 
   belongs_to :system
 
   has_many :translations
   has_many :pages
+
+  has_many :latest_translations
+
+  def latest_translations
+    Translation.find_by_sql("select T.* from translations T inner join (select language_id, resource_id, max(version) as
```

Comment:
I see the challenge in the problem you are trying to solve, and apart from caching the 'latest' version or using Postgres's "DISTINCT ON" plugin, I can't think of a better way of doing this, and neither of those stand out as preferable over this. Could you move this into the `Resource` class so it may be tested there and add some code comments as to how it works?